common/notification/index.ts

```diff
@@ -258,19 +259,24 @@ async function deliverNotification(
     };
 
     try {
-        const channel = channels.find((it) => it.canSend(notification));
-        if (!channel) {
+        const legacyUser = getUserForTypeORM(user);
```

Comment:
iirc, this might not be the legacy user but the newUser, just calling it user and the other one in the function signature legacyUser should be fine.
Please correct me if I'm wrong @Jonasdoubleyou.